quobyte_driver.go

```diff
@@ -38,11 +38,25 @@ func newQuobyteDriver(apiURL string, username string, password string, quobyteMo
 	return driver
 }
 
+func (driver quobyteDriver) StripVolumeName(requestName string) (strippedVolumeName string, subdirPath string) {
+	nameElems := strings.SplitN(requestName, "/", 2)
```

Comment:
We should use https://golang.org/pkg/path/#Clean on the `requestName` to clean up paths like `volumename/with/a//path`. Probably some input validation would also be nice.

Reply:
Added path cleaning.
regarding validation I'm still thinking. It probably makes sense to prevent ops above the mount point...

Reply:
I ran some tests and path.Clean already handles the parent dir topic gracefully so we'll stick with that for now.

Reply:
👍